bench/wtperf/wtperf.c

```diff
@@ -524,13 +529,19 @@ worker(void *arg)
 	WTPERF_THREAD *thread;
 	WT_CONNECTION *conn;
 	WT_CURSOR **cursors, *cursor, *log_table_cursor, *tmp_cursor;
+	WT_ITEM newv, oldv;
+	WT_MODIFY entries[MAX_NUM_OF_MODIFICATIONS];
 	WT_SESSION *session;
 	size_t i;
-	uint32_t total_table_count;
+	size_t iter;
+	size_t modify_offset;
+	size_t modify_size;
+	size_t total_modify_size;
+	uint32_t rand_val, total_table_count;
```

Comment:
Move this to below the 64-bit declarations for decreasing size order.

Reply:
corrected.